internal/pkg/cli/deploy/rdws.go

```diff
@@ -112,12 +112,17 @@ func (d *rdwsDeployer) DeployWorkload(in *DeployWorkloadInput) (ActionRecommende
 	if err != nil {
 		return nil, err
 	}
+	recommendations := &rdwsDeployOutput{
+		rdwsAlias: stackConfigOutput.rdSvcAlias,
+	}
 	if err := d.deploy(in.Options, stackConfigOutput.svcStackConfigurationOutput); err != nil {
+		var errStackUpdateCanceledOnInterrupt *cloudformation.ErrStackUpdateCanceledOnInterrupt
+		if errors.As(err, &errStackUpdateCanceledOnInterrupt) {
+			return recommendations, err
```

Comment:
Yes you are right. if I change alias in the manifest. This will display the wrong url in the recommended actions.
I will go with no recommendations in case of `cancel update stack` as well.